app/composables/useTimeTracker.ts

```
@@ -386,24 +389,21 @@
 }
 
 // Helper functions
-function getStartOfWeek(date: Date): Date {
-	const result = new Date(date)
-	const day = result.getDay()
-	const diff = result.getDate() - day
-	result.setDate(diff)
-	result.setHours(0, 0, 0, 0)
-	return result
+function getStartOfWeek(date: Temporal.PlainDate): Temporal.PlainDate {
```

ESLint

[error] 'getStartOfWeek' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)